casper/src/main/scala/coop/rchain/casper/Casper.scala

```diff
@@ -244,6 +238,42 @@ sealed abstract class MultiParentCasperInstances {
           maybeCheckPoint
         }
 
+      /*
+       * TODO: Put tuplespace validation back in after we have deterministic unforgeable names.
+       *
+       * We want to catch equivocations only after we confirm that the block completing
+       * the equivocation is otherwise valid.
+       */
+      private def attemptAdd(b: BlockMessage): F[BlockStatus] =
+        for {
+          dag                  <- Capture[F].capture { _blockDag.get }
+          postValidationStatus <- Validate.validateBlockSummary[F](b, genesis, dag)
+          // postTransactionsCheckStatus <- validateTransactions(...)
```

Comment:
Please remove old, commented out code

Reply:
No this is new code that we need to insert in the future

Reply:
Then maybe a TODO? Can be done in future commit though, not a PR blocker

Reply:
The TODO exists above the method